tests/qos/files/cisco/qos_param_generator.py

```diff
@@ -343,7 +343,7 @@ def __define_lossy_queue_voq(self):
                       "pg": 0,
                       "pkts_num_trig_egr_drp": self.max_depth // self.buffer_size,
                       "pkts_num_margin": 4,
-                      "packet_size": 64,
+                      "packet_size": 1350,
```

Comment:
This is a fix for the G100, but it looks like the Q200 will also use this change.
Did you run test for the Q200 to avoid regression?

Reply:
Yes, it passed on Q200.
```
============================================================================================= PASSES ==============================================================================================
_____________________________________________________________________ TestQosSai.testQosSaiLossyQueueVoqMultiSrc[single_asic] _____________________________________________________________________
---------------------------------------- generated xml file: /tmp/qos/test_qos_sai.py::TestQosSai::testQosSaiLossyQueueVoqMultiSrc_2024-08-16-16-57-18.xml ----------------------------------------
------------------------------------------------------------------------------------- live log sessionfinish --------------------------------------------------------------------------------------
17:04:49 __init__.pytest_terminal_summary         L0067 INFO   | Can not get Allure report URL. Please check logs
===================================================================================== short test summary info =====================================================================================
PASSED qos/test_qos_sai.py::TestQosSai::testQosSaiLossyQueueVoqMultiSrc[single_asic]
SKIPPED [1] qos/test_qos_sai.py:2219: Did not find any frontend node that is multi-asic - so can't run single_dut_multi_asic tests
SKIPPED [1] qos/test_qos_sai.py:2219: multi-dut is not supported on T1 topologies
======================================================================= 1 passed, 2 skipped, 1 warning in 448.29s (0:07:28) =======================================================================
INFO:root:Can not get Allure report URL. Please check logs
sonic@sonic-ucs-m3-4:/data/tests$ 
```
This run used packet size 1350:
```
17:00:48 base._run                                L0071 DEBUG  | /data/tests/ptf_runner.py::ptf_runner#95: [ptf_chu-1] AnsibleModule::shell, args=["/root/env-python3/bin/ptf --test-dir saitests/py3 sai_qos_tests.LossyQueueVoqMultiSrcTest --platform-dir ptftests --qlen=10000 --platform remote -t 'router_mac='\"'\"'9C:54:16:8B:70:00'\"'\"';src_server='\"'\"'1.74.25.72:9092'\"'\"';port_map_file='\"'\"'/root/ptf_test_port_map.json'\"'\"';sonic_asic_type='\"'\"'cisco-8000'\"'\"';sonic_version='\"'\"'azure_cisco_202405.14399-dirty-20240807.151042'\"'\"';src_dut_index=0;src_asic_index=0;dst_dut_index=0;dst_asic_index=0;dut_username='\"'\"'cisco'\"'\"';dut_password='\"'\"'cisco123'\"'\"';platform_asic='\"'\"'cisco-8000'\"'\"';dst_port_id=26;dst_port_ip='\"'\"'10.0.0.5'\"'\"';dst_port_vlan=None;dst_port_2_id=28;dst_port_2_ip='\"'\"'10.0.0.9'\"'\"';dst_port_2_vlan=None;dst_port_3_id=30;dst_port_3_ip='\"'\"'10.0.0.13'\"'\"';dst_port_3_vlan=None;src_port_id=24;src_port_ip='\"'\"'10.0.0.1'\"'\"';src_port_vlan=None;uplink_port_ids=[24, 26, 28, 30];uplink_port_ips=['\"'\"'10.0.0.1'\"'\"', '\"'\"'10.0.0.5'\"'\"', '\"'\"'10.0.0.9'\"'\"', '\"'\"'10.0.0.13'\"'\"'];uplink_port_names=['\"'\"'Ethernet96'\"'\"', '\"'\"'Ethernet112'\"'\"', '\"'\"'Ethernet128'\"'\"', '\"'\"'Ethernet144'\"'\"'];downlink_port_ids=[0, 1, 4, 5, 16, 17, 20, 21, 34, 37, 42, 43, 45, 46, 47, 50, 52, 53, 54, 55];downlink_port_ips=['\"'\"'10.0.0.33'\"'\"', '\"'\"'10.0.0.35'\"'\"', '\"'\"'10.0.0.37'\"'\"', '\"'\"'10.0.0.39'\"'\"', '\"'\"'10.0.0.41'\"'\"', '\"'\"'10.0.0.43'\"'\"', '\"'\"'10.0.0.45'\"'\"', '\"'\"'10.0.0.47'\"'\"', '\"'\"'10.0.0.49'\"'\"', '\"'\"'10.0.0.51'\"'\"', '\"'\"'10.0.0.53'\"'\"', '\"'\"'10.0.0.55'\"'\"', '\"'\"'10.0.0.57'\"'\"', '\"'\"'10.0.0.59'\"'\"', '\"'\"'10.0.0.61'\"'\"', '\"'\"'10.0.0.63'\"'\"', '\"'\"'10.0.0.65'\"'\"', '\"'\"'10.0.0.67'\"'\"', '\"'\"'10.0.0.69'\"'\"', '\"'\"'10.0.0.71'\"'\"'];downlink_port_names=['\"'\"'Ethernet0'\"'\"', '\"'\"'Ethernet4'\"'\"', '\"'\"'Ethernet16'\"'\"', '\"'\"'Ethernet20'\"'\"', '\"'\"'Ethernet64'\"'\"', '\"'\"'Ethernet68'\"'\"', '\"'\"'Ethernet80'\"'\"', '\"'\"'Ethernet84'\"'\"', '\"'\"'Ethernet168'\"'\"', '\"'\"'Ethernet180'\"'\"', '\"'\"'Ethernet200'\"'\"', '\"'\"'Ethernet204'\"'\"', '\"'\"'Ethernet212'\"'\"', '\"'\"'Ethernet216'\"'\"', '\"'\"'Ethernet220'\"'\"', '\"'\"'Ethernet232'\"'\"', '\"'\"'Ethernet240'\"'\"', '\"'\"'Ethernet244'\"'\"', '\"'\"'Ethernet248'\"'\"', '\"'\"'Ethernet252'\"'\"'];testPortIds={0: {0: [0, 1, 4, 5, 16, 17, 20, 21, 24, 26, 28, 30, 34, 37, 42, 43, 45, 46, 47, 50, 52, 53, 54, 55]}};testPortIps={0: {0: {0: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.33'\"'\"'}, 24: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.1'\"'\"'}, 1: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.35'\"'\"'}, 4: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.37'\"'\"'}, 26: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.5'\"'\"'}, 5: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.39'\"'\"'}, 16: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.41'\"'\"'}, 28: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.9'\"'\"'}, 17: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.43'\"'\"'}, 20: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.45'\"'\"'}, 30: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.13'\"'\"'}, 21: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.47'\"'\"'}, 34: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.49'\"'\"'}, 37: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.51'\"'\"'}, 42: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.53'\"'\"'}, 43: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.55'\"'\"'}, 45: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.57'\"'\"'}, 46: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.59'\"'\"'}, 47: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.61'\"'\"'}, 50: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.63'\"'\"'}, 52: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.65'\"'\"'}, 53: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.67'\"'\"'}, 54: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.69'\"'\"'}, 55: {'\"'\"'peer_addr'\"'\"': '\"'\"'10.0.0.71'\"'\"'}}}};testbed_type='\"'\"'t1-56-lag'\"'\"';dscp=8;ecn=1;pg=0;src_port_2_id=0;src_port_2_ip='\"'\"'10.0.0.33'\"'\"';pkts_num_leak_out=0;pkts_num_trig_egr_drp=16000;packet_size=1350;cell_size=384;pkts_num_margin=4' --debug info --log-file /tmp/sai_qos_tests.LossyQueueVoqMultiSrcTest.log --socket-recv-size 16384 --test-case-timeout 1200  --disable-ipv6 --disable-vxlan --disable-geneve --disable-erspan --disable-mpls --disable-nvgre"], kwargs={"chdir": "/root", "module_ignore_errors": false, "module_async": false}
```